operators/Cargo.toml

```diff
@@ -50,6 +50,8 @@ tracing = "0.1"
 tracing-opentelemetry = "0.18"
 typetag = "0.2"
 uuid = { version = "1.1", features = ["serde", "v4", "v5"] }
+xgboost = {git = "https://github.com/MarWeUMR/xgboost-rs", tag = "v0.1.0"}
+ndarray = "*"
```

Comment:
please specify the version and put it under dev-dependencies, as it is only used for tests

Reply:
https://github.com/geo-engine/geoengine/pull/639/commits/655449c29c8578e7b27cdd6c4579ead07c50b880

Reply:
we usually don't specify the concrete patch. Just set it to "0.15"